core/templates/pages/exploration-editor-page/translation-tab/modal-templates/add-audio-translation-modal.template.html

```diff
@@ -9,11 +9,11 @@ <h3>Add Audio Translation</h3>
         <div class="form-group">
           <label for="audioTranslationFileField" class="col-lg-2 col-md-2 col-sm-2 float-left">Audio File</label>
           <div class="col-lg-10 col-md-10 col-sm-10">
-            <audio-file-uploader id="audioTranslationFileField"
-                                 dropped-file="droppedFile"
-                                 on-file-changed="updateUploadedFile"
-                                 on-file-cleared="clearUploadedFile">
-            </audio-file-uploader>
+            <oppia-audio-file-uploader id="audioTranslationFileField"
+                                       [dropped-file]="droppedFile"
+                                       (change-file)="updateUploadedFile($event)"
```

Comment:
When naming `@Outputs` think of the name with `on`. In this case, I think of the name as `on-file-change`. Once you've thought of it, drop the `on` prefix.

Reply:
Thanks for the suggestion, done.